registration_ref/app.py

```diff
@@ -46,8 +46,14 @@ def create_in_foundries(client_cert: str, api_token: str, name: Optional[str] =
         "OSF-TOKEN": api_token,
     }
     for x in (0.1, 0.2, 1, 0):
+        log.info("Initiating request")
+        log.info("Using DEVICE_REGISTRATION_API: %s", Settings.DEVICE_REGISTRATION_API)
+        log.info("Using VERIFY_SSL: %s", Settings.VERIFY_SSL)
```

Comment:
maybe do this as one log entry instead of 3. The logs for a real server with many devices would just be unreadable - and there's no context to tie it a device to actually help

Reply:
It seems that each one of us has a preference regarding the logs.
So, I decided to remove it to make it easier to convey and let us move forward. We can try to improve that in a follow-up.

Reply:
@camilamacedo86 did you forget to force-push your changes? I still see these logs here.